numba/typeinfer.py

```diff
@@ -1323,8 +1323,20 @@ def typeof_global(self, inst, target, gvar):
                 # as a global variable
                 typ = types.Dispatcher(_temporary_dispatcher_map[gvar.name])
             else:
-                msg = _termcolor.errmsg("Untyped global name '%s':") + " %s"
-                e.patch_message(msg % (gvar.name, e))
+                nm = gvar.name
+                msg = _termcolor.errmsg("Untyped global name '%s':" % nm)
+                msg += " %s" # interps the actual error
+
+                # if the untyped global is a numba internal function then add
+                # to the error message asking if it's been imported.
+                from numba import special
+                numba_internals = special.__all__[1:]
+                if nm in numba_internals:
+                    tmp = ("\n'%s' is a Numba internal function, has it been "
```

Comment:
Replace `%s is a Numba internal function` with something less certain.  `%s looks like a Numba internal...`?

Reply:
Good idea.

Reply:
d038d8e fixes